generate-types.js

```diff
@@ -0,0 +1,60 @@
+const TypeWriter = require('@gimenete/type-writer')
+const webhooks = require('@octokit/webhooks-definitions/index.json')
+const pascalCase = require('pascal-case')
+const prettier = require('prettier')
+const fs = require('fs')
+
+const types = []
+const signatures = []
+for (const webhook of webhooks) {
+  const { name, actions, examples } = webhook
+  if (!examples) continue
+  const tw = new TypeWriter()
+  const typeName = `WebhookPayload${pascalCase(name)}`
+  tw.add(examples, { rootTypeName: typeName })
+  const events = [
+    `'${name}'`,
+    ...actions.map(action => `'${name}.${action}'`)
+  ].join(' | ')
+  types.push(`type ${typeName} = ${tw.generate('typescript', { inlined: true })}`)
+  signatures.push(`
+    public on (event: ${events}, callback: (event: Webhooks.WebhookEvent<${typeName}>) => void)
+  `)
+}
+
+const definition = `
+// import { Application } from 'express'
```

Comment:
We don’t use express, is this obsolete?

Reply:
That was copied from https://github.com/probot/probot/blob/master/src/%40types/%40octokit/webhooks/index.d.ts

I commented it out because maybe probot needs it. But now I thinking that if needed probot can just extend the type. So I'll delete that.